marquez_client/marquez.py

```diff
@@ -13,16 +13,14 @@
 import logging
```

Comment:
the dir / fiel naming here is a little weird -- can we do something like name this directory `marquez` and this file `client.py`?

Reply:
Sure!